package.json

```diff
@@ -32,7 +31,7 @@
     "coverage": "nyc --reporter html tape test/*.test.js && opener coverage/index.html"
   },
   "devDependencies": {
-    "aws-sdk-mock": "1.3.0",
+    "@mapbox/mock-aws-sdk-js": "0.0.4",
```

Comment:
There's a v0.0.5 now, does an upgrade still work here?

Reply:
`@mapbox/mock-aws-sdk-js: 0.0.5` needs `aws-sdk-js: 2.154.0` and throws lots of errors in tests which I think exceeds the scope of this PR. Will release as-is for now.